bazel/pl_workspace.bzl

```diff
@@ -82,3 +84,18 @@ def pl_model_files():
         downloaded_file_path = "sentencepiece.proto",
         sha256 = "7e17e04ecc207d9204dc8755357f988bf77c135f7a34a88984943c8649d6a790",
     )
+
+def qemu_with_kernel_deps():
+    http_file(
+        name = "linux_build_5_18_19_x86_64",
```

Comment:
It's unclear from the name what this actually is. Is it the `bzImage`? If so can you call it `linux_bzImage_...`

Reply:
Talked offline, this name is fine since its more than just bzImage